svc_translation.go

```diff
@@ -222,10 +225,12 @@ type TranslationListOptions struct {
 	// Possible values are 1 and 0.
 	DisableReferences uint8 `url:"disable_references,omitempty"`
 
-	FilterLangID     string `url:"filter_lang_id,omitempty"`
-	FilterIsReviewed uint8  `url:"filter_is_reviewed,omitempty"`
-	FilterFuzzy      uint8  `url:"filter_fuzzy,omitempty"`
-	FilterQAIssues   string `url:"filter_qa_issues,omitempty"`
+	FilterLangID       int64  `url:"filter_lang_id,omitempty"`
```

Comment:
Breaking changes

Reply:
Yes, because the current version is invalid and won't work with Lokalise Api